### PR TITLE
Added wallet library package version to all the network calls

### PR DESCRIPTION
**Pull request by nithyaganeshng**
Sends the platform name and version of Wallet Library package being used as headers on all the network calls.

**Validation:**
Please include here how it was verified that the PR works as intended.

**Type of change:**
- [x] Feature work
- [ ] Bug fix
- [ ] Documentation
- [ ] Engineering change
- [ ] Test
- [ ] Logging/Telemetry

**Risk**:
- [ ] High – Errors could cause MAJOR regression of many scenarios. (Example: new large features or high level infrastructure changes)
- [ ] Medium – Errors could cause regression of 1 or more scenarios. (Example: somewhat complex bug fixes, small new features)
- [x] Small – No issues are expected. (Example: Very small bug fixes, string changes, or configuration settings changes)

**Work Item links**:
Please include here links for this work item, or deferred work, or related work. E.g. if the refactoring is too big to fit in this PR, or the localized strings need to be updated later, please link the TODO work items here.

**Documentation Links**:
Please include here links to any related background documentation for this PR.

### CODE REVIEW

File: walletlibrary/build.gradle [hunk 1]
```diff
@@ -21,16 +23,21 @@ android {
...
+        versionName _version
+        buildConfigField 'String', 'WalletLibraryVersion', "\"$_version\""
 
         testInstrumentationRunner "androidx.test.runner.AndroidJUnitRunner"
     }
 
     buildTypes {
         release {
-            minifyEnabled false
+            minifyEnabled true
```
Comment (symorton) [inline walletlibrary/build.gradle]: What is minifyEnabled for?

File: walletlibrary/src/main/java/com/microsoft/did/sdk/VerifiableCredentialSdk.kt [hunk 2]
```diff
@@ -67,11 +67,13 @@ internal object VerifiableCredentialSdk {
         logConsumer: SdkLog.Consumer = DefaultLogConsumer(),
         polymorphicJsonSerializers: SerializersModule = Json.serializersModule,
         registrationUrl: String = "",
-        resolverUrl: String = "https://discover.did.msidentity.com/v1.0/identifiers"
+        resolverUrl: String = "https://discover.did.msidentity.com/v1.0/identifiers",
+        walletLibraryVersionInfo: String = ""
```
Comment (symorton) [inline walletlibrary/src/main/java/com/microsoft/did/sdk/VerifiableCredentialSdk.kt:72]: Is this temporary? Should we add a TODO to remember to change how this is implemented once we implement a new HTTP Client layer?

**Review by symorton — CHANGES_REQUESTED** [review]

File: walletlibrary/src/main/java/com/microsoft/did/sdk/VerifiableCredentialSdk.kt (hunk 2 above)
Comment (nithyaganeshng) [reply walletlibrary/src/main/java/com/microsoft/did/sdk/VerifiableCredentialSdk.kt:72]: Added a TODO.

File: walletlibrary/build.gradle (hunk 1 above)
Comment (nithyaganeshng) [reply walletlibrary/build.gradle]: I will do some more testing before I turn it on. It would reduce the size of APK by removing the unused code.

**Review by logangirvin — DISMISSED** [review]

**Review by symorton — DISMISSED** [review]

**Review by logangirvin — APPROVED** [review]

**Pull request merged**